clouddriver-sql/src/main/resources/db/changelog/20190822-initial-event-schema.yml

```diff
@@ -2,6 +2,7 @@ databaseChangeLog:
   - changeSet:
       id: create-event-aggregates-table
       author: robzienert
+      validCheckSum: 8:99ad0fa01e0a2f5a2d66e015a58dccfd
```

Comment:
What happens for users running MySQL and then upgrading to this new version? Is this `initial-event-schema` applied again? I guess it should not be applied and existing installations will still have the `bigint(20)` constraint, but new ones will not be constrained.

Reply:
Correct. `bigint(20)` is the same as `bigint` with a hint of the size of the data for the client.